### ISSUE TITLE
Suggestion: Include Import ID in fetch_survey tibble

### ISSUE THREAD

**Issue by ashbaldry** [ropensci/qualtRics#116]
Hi,

Just come across this package and like how easy it is to use. Would it be possible to add the import IDs somewhere in the `fetch_survey()` output?

I have pulled a couple of our surveys and we have quite a few duplicate questions. I know the second question will have the `_1` suffix in the fetch_survey column names, but for future proofing the mapping of the questions to the responses it would be good to have this added in as I've found the question ids to change but the import ids stay consistent.

I do have a way to extract them within `read_survey()` but didn't know where best to store them in the tibble.

**Comment by juliasilge** [comment]
Thanks for suggesting this @ashbaldry! Can you share with me how you extracted the specific IDs you are interested in using `read_survey()`? I wonder if this is something none of my surveys have, because I'm not 100% sure what you're referring to. Some example code and output would be very helpful!

**Comment by ashbaldry** [comment]
Sure! The values are pretty much what are available in the `qid` column from `survey_questions()`. As an example of where I do get duplicates, this is the top three rows of my survey within `read_survey()`.
```
suppressMessages(readr::read_csv(
  file = file_name,
  col_names = FALSE,
  n_max = 3
))

  X97                                       X104
  <chr>                                     <chr>
1 Q3.14_Q3.14_12                            Q3.14_Q3.14_12
2 To what extent would you agree with...    To what extent would you agree with...
3 {'ImportId': 'QID15-12'}                  {'ImportId': 'QID104-12'} 
```

So this is what I am using to get the import ids from the csv:
```
imports <- suppressMessages(readr::read_csv(
  file = file_name,
  col_names = FALSE,
  skip = skipNr - 1,
  n_max = 1
))

importrow <- unlist(imports)
importids <- gsub("^\\{'ImportId': '(.*)'\\}$", "\\1", importrow)

importids
[1] "QID15-12"  "QID104-12"
```

**Comment by juliasilge** [comment]
OK, I found the specific IDs you are referring to and I can see how they would be useful. 

Would you find it useful to have an option to get these IDs as your column names **instead** of the current column names? I am picturing something like `importID = FALSE` as a default, and then if you change it to `TRUE`, you get the import IDs as column names instead of whatever Qualtrics sends over now.

**Comment by ashbaldry** [comment]
That would definitely be useful :) Think the `importID` flag is much cleaner than trying to add them in somewhere awkwardly in the tibble.

**Comment by juliasilge** [comment]
I have this implemented now in commit b595d34a1e9c5f544a42930c88dd6aee9c55b5c9. Give it a try and see if you have any feedback! ✨

**Comment by ashbaldry** [comment]
Just tried it, and it works great, thanks!
